tdrs-frontend/cypress/e2e/common-steps/common-steps.js

```diff
@@ -1,38 +1,44 @@
 import { When } from '@badeball/cypress-cucumber-preprocessor'
 
 When('{string} visits the home page', (username) => {
```

Comment:
can't we store the admin and user separately and use them from store as necessary, instead of deleting them?

Reply:
@jtimpe 🆙

Reply:
this is technically what's being done already via the following calls in `cypress/support/commands.js`

```js
// regular login
cy.getCookie('sessionid').its('value').as('userSessionId')
cy.getCookie('csrftoken').its('value').as('userCsrfToken')

// admin login
cy.getCookie('sessionid').its('value').as('adminSessionId')
cy.getCookie('csrftoken').its('value').as('adminCsrfToken')
```

this takes the value from the cookie immediately after login and stores it in the cypress intermediary variable, which lives for the test run. So the `cy.clearCookie` and `cy.setCookie` apply to the browser window during the test run and use the stored values.

We should do better caching of the vars so that we don't have to re-login both users before *every* test, but that's covered in a followup ticket #3274